src/ip/Intercept.h

```diff
@@ -30,8 +30,8 @@ class Intercept
     Intercept() : transparentActive_(0), interceptActive_(0) {}
     ~Intercept() {};
 
-    /** Perform NAT lookups */
-    bool Lookup(const Comm::ConnectionPointer &newConn, const Comm::ConnectionPointer &listenConn);
+    /// use NAT to set the local address of the given freshly accepted connection
```

Comment:
But perhaps we should mention that these lookups affect the 'local' part of the Connection? E.g.,:
```suggestion
    /// Perform NAT lookups for the local address of the given connection
```
Or is this already implied, since we cannot lookup remote address?

Reply:
I am okay with that.